roles/os_zabbix/library/zbxapi.py

```diff
@@ -69,23 +84,25 @@ class ZabbixAPI(object):
         'Usermedia': ['get'],
     }
 
-    def __init__(self, data={}):
-        self.server = data['server'] or None
-        self.username = data['user'] or None
-        self.password = data['password'] or None
-        if any(map(lambda value: value == None, [self.server, self.username, self.password])):
+    def __init__(self, data=None):
+        if not data:
+            data = {}
```

Comment:
In the comments in the blog you bosted, Ned states that Alex's method (same as used in my code) is the safest way to handle passed variables.